client/my-sites/people/role-select/index.jsx

```diff
@@ -76,6 +76,7 @@ const RoleSelect = ( props ) => {
 	);
 };
 
+export { RoleSelect };
```

Comment:
this is suspicious ^^^ - if we need to export `RoleSelect` we should probably do it on the line where `RoleSelect` is defined

```js
export const RoleSelect = …
```